BUILD.bazel

```diff
@@ -22,6 +38,14 @@ cc_library(
     hdrs = ["include/benchmark/benchmark.h"],
     linkopts = select({
         ":windows": ["-DEFAULTLIB:shlwapi.lib"],
+        ":freebsd": [
```

Comment:
if we `load("@bazel_skylib//lib:selects.bzl", "selects")` then we can replace this with:

```
linkopts = selects.with_or({
    ":windows": ["-DEFAULTLIB:shlwapi.lib"],
    (":freebsd", ":openbsd"): ["-pthread", "-lm"],
})
```

which i think is going to be easier to read/maintain. assuming freebsd/openbsd don't diverge.

as an alternative, you can define:

```
selects.config_setting_group(
    name = "bsd",
    match_any = [":freebsd", ":openbsd"],
)
```

and then just use ":bsd" as the key here.

up to you :)

Reply:
That would work within the repository, but then it requires every dependent repository to install @bazel_skylib too, I guess. It turns out that Bazel doesn't load `WORKSPACE` dependencies transitively (?), and the "@platforms" package appears to load implicitly. I'm wondering whether these Bazel utility repositories should be built-ins or be loaded by default.

I've done something simpler: defined a variable named `BSD_LINKOPTS` and shared it with both the ":freebsd" and ":openbsd" keys. Would that work for you?

Reply:
I've filed an [issue](https://github.com/bazelbuild/bazel/issues/12014) with the Bazel team. I like the approach you've suggested, but adding a dependency here would complicate things for users, which I'm not so sure they'd like.

Reply:
that's a great compromise. thanks!